src/Reflectivity/MetaLink.class.st

```diff
@@ -477,16 +477,29 @@ MetaLink >> permaLinkFor: aClassOrObject option: option instanceSpecific: instan
 	^ permaLink
 ]
 
-{ #category : #'link api' }
+{ #category : #printing }
 MetaLink >> printString [
```

Comment:
I just noticed that it is #printString method instead of #printOn: .  Normally classes implement #printOn: with a stream as argument.

Reply:
Yes, I do not know why it was done like this @MarcusDenker 
It is not part of the PR (I only changed the method and its protocol).
Can you add an issue?

Reply:
yes, should be fixed.